README.md

```diff
@@ -218,6 +218,147 @@ You should create and keep your models under this folder. Note that we have appl
 
 Once your model is ready to submit, simply create a pull request on the github repo and let us know in our [Discord](https://discord.com/invite/BJbYSRDdtr).
 
+
+# SDK Command Documentation 
+## Help command
+
+All the commands accept `-h` parameter for help, e.g.:
+
+```
+>credmark-dev -h
+
+usage: credmark-dev [-h] [--log_level LOG_LEVEL] [--model_path MODEL_PATH] [--manifest_file MANIFEST_FILE]
+                    {list,list-models,build,build-manifest,clean,remove-manifest,run,run-model} ...
+
+Credmark developer tool
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --log_level LOG_LEVEL
+                        [OPTIONAL] Log level: DEBUG, INFO, WARNING, ERROR, CRITICAL
+  --model_path MODEL_PATH
+                        [OPTIONAL] Semicolon separated paths to the model folders (or parent) or model python file. Defaults to models folder.
+  --manifest_file MANIFEST_FILE
+                        [OPTIONAL] Name of the built manifest file. Defaults to models.yaml. (Not required during development.)
+
+Commands:
+  Supported commands
+
+  {list,list-models,build,build-manifest,clean,remove-manifest,run,run-model}
+                        
+additional help
+    list (list-models)  List models
+    build (build-manifest)
+                        Build model manifest
+    clean (remove-manifest)
+                        Clean model manifest
+    run (run-model)     Run a model
+```
+## `run`command
+Below -h command shows the details of options available for run commands.
+```
+>credmark-dev run -h
+
+usage: credmark-dev run [-h] [-b BLOCK_NUMBER] [-c CHAIN_ID] [-i INPUT] [-v MODEL_VERSION] [--provider_url_map PROVIDER_URL_MAP] [--api_url API_URL] model-slug
+
+positional arguments:
+  model-slug            Slug for the model to run.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -b BLOCK_NUMBER, --block_number BLOCK_NUMBER
+                        Block number used for the context of the model run. If not specified, it is set to the latest block of the chain.
+  -c CHAIN_ID, --chain_id CHAIN_ID
+                        [OPTIONAL] The chain ID. Defaults to 1.
+  -i INPUT, --input INPUT
+                        [OPTIONAL] Input JSON or if value is "-" it will read input JSON from stdin.
+  -v MODEL_VERSION, --model_version MODEL_VERSION
+                        [OPTIONAL] Version of the model to run. Defaults to latest.
+  --provider_url_map PROVIDER_URL_MAP
+                        [OPTIONAL] JSON object of chain id to Web3 provider HTTP URL. Overrides settings in env vars.
+  --api_url API_URL     [OPTIONAL] Credmark API url. Defaults to the standard API gateway. You do not normally need to set this.
+  --api_url API_URL     [OPTIONAL] Credmark API url
+```
+
+To call any model we need to provide below parameters:
```

Comment:
Maybe change this to something about these are the important parameters (they're not necessarily required.)